rllib/utils/policy.py

```diff
@@ -45,12 +45,13 @@ def validate_policy_id(policy_id: str, error: bool = False) -> None:
         ValueError: If the given `policy_id` is not a valid one and `error` is True.
     """
     if (
-        len(policy_id) == 0
+        not isinstance(policy_id, str)
```

Comment:
Bug fix. The old utility function would actually crash (unexpected) when being passed a non-string ID.